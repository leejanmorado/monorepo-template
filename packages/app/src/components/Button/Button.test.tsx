import { describe, it, expect } from 'vitest';
import { render } from '@testing-library/react';
import { Button } from '@components';

describe('Button component', () => {
	it('matches DOM snapshot', () => {
		const { container } = render(<Button>Click me</Button>);

		expect(container.firstChild).toMatchSnapshot();
	});
});
