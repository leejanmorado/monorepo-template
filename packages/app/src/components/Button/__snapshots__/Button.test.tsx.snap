// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Button component > matches DOM snapshot 1`] = `
<div
  class="h-10"
>
  <button
    class="rounded-md bg-white px-4 py-2 text-base text-black transition-all hover:bg-gray-300 active:bg-gray-300 active:py-[6px] active:px-[14px] active:text-sm  undefined"
  >
    Click me
  </button>
</div>
`;
